src/jaxsim/api/ode_data.py

```diff
@@ -169,24 +182,30 @@ def build_from_jaxsim_model(
                 base_linear_velocity=base_linear_velocity,
                 base_angular_velocity=base_angular_velocity,
             ),
-            soft_contacts_state=SoftContactsState.build_from_jaxsim_model(
+            contacts_state=getattr(
+                importlib.import_module(f"jaxsim.api.{module_name}"), class_name
+            ).build_from_jaxsim_model(
```

Comment:
If you import the supported contact models above as suggested, you can remove this piece of code that, in any case, would not support user-defined contact models (outside `jaxsim`).

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/178/commits/9675c4965004291dd3d78cae1546ff539b9e3f80